mixitup/csv_json.py

```diff
@@ -0,0 +1,20 @@
+import csv, ujson as json, pprint
```

Comment:
With this in mind, I have a couple of thoughts:

1. I think it is worth our time making a CDS StyleSheet that gives our team central guides to follow like:
     a. CSS selectors use name-with-dash
     b. Python Scripts are in their own folder on the root
     c. CSV, JSON, and other data files are in a data folder (or do they go in with JS?)
     d. Only one image folder per project 
and etc - does this make sense or is there some guideline document that already exists that you would like us to follow?

2. What is the answer to bullet "c"?